tests/MessageTest.php

```diff
@@ -394,13 +394,26 @@ public function an_exception_will_be_throw_in_invalid_string(): void
     }
 
     /** @test */
-    public function segment_ending_bar_can_be_omitted(): void
```

Comment:
Let's retain this test as long as we support `SEGMENT_ENDING_BAR`. Add your test as a new one please